grails-forge-core/src/main/java/org/grails/forge/build/gradle/GradleDependency.java

```diff
@@ -99,14 +101,15 @@ public int hashCode() {
 
     @NonNull
     public String toSnippet() {
-        String snippet = gradleConfiguration.getConfigurationName();
+        String snippet = gradleConfiguration.getConfigurationName() + " ";
         if (isPom()) {
-            String platformPrefix = " ";
-            snippet += platformPrefix + "platform";
+            snippet += "platform(";
+        } else if (gradleConfiguration == INTEGRATION_TEST_IMPLEMENTATION_TEST_FIXTURES) {
```

Comment:
If we ever add another Gradle scope that requires () we will re-engineer this code at that time.